source/ftrack_api/event/hub.py

```diff
@@ -855,12 +860,16 @@ def _get_socket_io_session(self):
             self.get_network_location()
         )
         try:
+            req_headers = {
+                'ftrack-user': self._api_user,
+                'ftrack-api-key': self._api_key
+            }
+            if self._headers:
+                req_headers.update(self._headers)
             response = requests.get(
```

Comment:
Should we pass in the session to the EventHub object? Right now, it's launching requests using the requests module, not our session object.